decidim/decidim
# Redesign: add button for mobiles in online meetings with iframes

#### :tophat: What? Why?
The iframe embed causes issues in terms of dimensions used on mobile as seen with previous PR's such as #11656. This PR will tackle the issue by altering the behaviour and giving the user a link to open their meeting instead.

Based on the agreed route for this fix with the @decidim/maintainers, the link will only be present if the minimum  dimensions are met on mobile by embedding the link with the tailwind class ```md:hidden```. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11358
- Relates https://github.com/decidim/decidim/pull/11656
- Relates https://github.com/decidim/decidim/pull/11840

#### Testing
1. Login and create a meeting 
2. Make sure the meeting is set in the 'past' and ends in the 'future' from the day you test.
3. Add a meeting link like 'jit.si' save and open the meeting in the browser.
4. In browser find your meeting page and click inspect
5. Change the dimensions to mobile
6. make it smaller to see the link appear.

### :camera: Screenshots
With the display block (embed) only shown if the tailwind class ```md``` break point is met:
![Screenshot 2023-10-31 at 11 32 57](https://github.com/decidim/decidim/assets/101816158/84487dbb-9228-4720-a094-8912296b5577)

The link displayed when the breakpoint ```md``` dimensions are surpassed as per requirement:
![Screenshot 2023-10-31 at 11 13 41](https://github.com/decidim/decidim/assets/101816158/60d251b1-9127-4c05-8c64-13bfeeb9a1a6)


:hearts: Thank you!
